nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -1880,6 +1887,17 @@ impl<'a> BlueprintBuilder<'a> {
             .len()
     }
 
+    /// Set the `target_release_minimum_generation` field for this blueprint.
+    pub fn set_target_release_minimum_generation(
+        &mut self,
+        target_release_minimum_generation: Generation,
+    ) -> Result<(), Error> {
+        // May need to do some validation here in the future.
```

Comment:
I think there should be a compare-and-swap like operation here to catch concurrent updates. I understand this would be rare, given it's a human making the modifications via omdb, but we do something similar with clickhouse generations.

Would it perhaps be better to have a human specify the current generation they expect and then increment the generation if it matches?

Reply:
I made this function compare-and-set, and then took care of that behind the scenes in reconfigurator-cli :)